tests/integration/s2n_client_endpoint_handshake_test.py

```diff
@@ -114,15 +122,22 @@ def well_known_endpoints_test(use_corked_io):
 
 def main(argv):
     if len(argv) < 2:
-        print("s2n_handshake_test_s_client.py host port")
+        print("s2n_client_endpoint_handshake_test.py host port")
```

Comment:
nice catch. don't think this file uses any host/port arguments tho

Reply:
I know using argparse seems overkill for just two arguments, but it does handle the help menu very nicely.

Reply:
Argparse is definitely the preferred approach to command line arguments, and Josh makes a good point, host and port aren't actually used anywhere in the test. I've replaced this structure with argparse and removed the host and port arguments from the test and calling makeile. I also added an option to disable TLS 1.3 tests, but they're on by default.